src/plots/rectanglePlot.ts

```diff
@@ -303,6 +305,50 @@ export module Plots {
       return intersected;
     }
 
+    /**
+     * Gets the Accessor for label.
+     */
+    public label(): Accessor<string>;
+    /**
+     * Sets label to the result of an Accessor.
+     *
+     * @param {Accessor<string>} label
+     * @returns {Plots.Rectangle} The calling Rectangle Plot.
+     */
+    public label(label: Accessor<string>): Plots.Rectangle<X, Y>;
+    public label(label?: Accessor<string>): any {
+      if (label == null) {
+        return this._label;
+      }
+
+      this._label = label;
+      this.render();
+      return this;
+    }
+
+    /**
+     * Get whether rectangle labels are enabled.
```

Comment:
"Gets whether labels are enabled."

 Also, on getters for simple values the description after the `@returns` can be safely omitted.